docs/REGISTRY-SETUP.md

```
@@ -0,0 +1,313 @@
+# Private Container Registry Setup Guide
+
+## Overview
+
+This guide covers setting up a self-hosted Harbor registry as a proxy cache for hardened container images from multiple sources (dhi.io, cgr.dev, gcr.io).
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)